packages/rspack/tests/diagnostics/module-parse-failed/non_support_warning/stats.err

```diff
@@ -0,0 +1,9 @@
+WARNING in ./index.js
+ModuleParseWarning
+
+  ⚠ Module parse warning:
+  ╰─▶   ⚠ JavaScript parsing error: require.include() is not supported by Rspack.
```

Comment:
Please avoid full expression, instead, you can use:
```suggestion
  ╰─▶   ⚠ JavaScript parsing error: `require.include` is not supported by Rspack.
```

Reply:
This is not full expression, you can see the code here. Add `()`means that it is a call expression